### PR TITLE
Finnish locale for staticpages plugin

### PR DESCRIPTION
**Pull request by ajnyga**
Hi @mtub 

Fi locale for the plugin...

### CODE REVIEW

File: locale/fi_FI/locale.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/en_US/locale.xml
```
Comment (mtub) [inline locale/fi_FI/locale.xml]: locale/fi_FI/locale.xml

File: locale/fi_FI/locale.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
```
Comment (mtub) [inline locale/fi_FI/locale.xml]: ... 2017

File: locale/fi_FI/locale.xml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  * Copyright (c) 2003-2016 John Willinsky
```
Comment (mtub) [inline locale/fi_FI/locale.xml]: ... -2017

**Review by mtub — CHANGES_REQUESTED** [review]
please update according to my comments…

**Comment by ajnyga** [comment]
check!

**Pull request merged**